### PR TITLE
Update recruiting.md

### PR DESCRIPTION
**Pull request by natalie-hub**
Add referral bonus

### CODE REVIEW

File: contributors/join-us/recruiting.md [hunk 1]
```diff
@@ -24,6 +24,28 @@ We currently have staff in the following U.S. states:
 
 Preparing to hire an employee in a new state takes time, and we may be subject to new regulations, taxes, and reporting obligations. If a candidate is progressing in the interview process and would be living and working from a U.S. state not listed above, the Recruiter will DM @tim.quock with the state and the title of the position the candidate may fill at least three weeks prior to an anticipated offer.
 
+### Referral Program
+
+Mattermost Staff and Community members are eligible to receive a $500 USD referral bonus after the candidate they refer is hired and completes 90 days of service with Mattermost.
```
Comment (justinegeffen) [inline contributors/join-us/recruiting.md]:
```suggestion
Mattermost Staff and Community members are eligible to receive a $500 USD referral bonus if the candidate they refer is hired and completes 90 days of service with Mattermost.
```

File: contributors/join-us/recruiting.md [hunk 2]
```diff
@@ -24,6 +24,28 @@ We currently have staff in the following U.S. states:
...
+
+**Referral Criteria**
```
Comment (justinegeffen) [inline contributors/join-us/recruiting.md]:
```suggestion
**Referral criteria**
```

**Comment by justinegeffen** [comment]
Thanks, @natalie-hub! Just one comment, otherwise looks good to me!

**Review by justinegeffen — DISMISSED** [review]

File: contributors/join-us/recruiting.md (hunk 1 above)
Comment (natalie-hub) [reply contributors/join-us/recruiting.md]: Thank you, as always, @justinegeffen :)

**Review by natalie-hub — COMMENTED** [review]
Thanks for the changes!

**Review by natalie-hub — COMMENTED** [review]
Thank you!

**Comment by natalie-hub** [comment]
@justinegeffen - it says I dismissed your "stale review." I'm not sure what "stale" means, but if I'm OK with your suggested changes. Did I mess something up? Can you let me know how I "dismissed" it so I know to avoid next time? Sorry - such a GitHub noob.

**Comment by justinegeffen** [comment]
No worries at all, @natalie-hub. That does sometimes happen but it doesn't affect the content. :)

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**